packages/backend-for-frontend/src/routers/catalogRouter.ts

```diff
@@ -968,18 +968,43 @@ const catalogRouter = (
       }
     })
     .post(
-      "/templates/eservices/:eServiceId/descriptors/:descriptorId/interface",
+      "/templates/eservices/:eServiceId/descriptors/:descriptorId/interface/soap",
       async (req, res) => {
         const ctx = fromBffAppContext(req.ctx, req.headers);
         try {
-          const response =
-            await eserviceTemplateService.addEserviceInterfaceByTemplate(
+          const eserviceId =
```

Comment:
This should not be the `eserviceId` but rather the id of the created resource, which in this case should be the new document id

Reply:
this commit [d1e9dde](https://github.com/pagopa/interop-be-monorepo/pull/1643/commits/d1e9dde2510d1e7615a163496e73612e3c097faa) contains this refactor